src/components/Header.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FormEventHandler, useContext, useEffect, useRef } from 'react';
+import '../styles/todoapp.scss';
+import { CreatedContext } from './ToDoContext';
+import classNames from 'classnames';
+
+export const Header: React.FC = () => {
+  const { todos, setTodos, toDoTitle, setToDoTitle } =
+    useContext(CreatedContext);
+  const handleInputSubmit: FormEventHandler = event => {
+    event.preventDefault();
+    const newTodo = {
+      id: Date.now(),
+      title: toDoTitle.trim(),
+      completed: false,
+    };
+
+    if (!toDoTitle.trim()) {
+      return;
+    }
+
+    setTodos([...todos, newTodo]);
```

Comment:
everywhere if you want to update your state based on prev state use functional setState